### ISSUE TITLE
NTA MOE should not appear for top-coded values

### ISSUE THREAD

**Issue by EricaMaurer** [NYCPlanning/labs-factfinder#759]
For NTA geographies (from database) MOE should not appear for top-coded values. Showing in PFF as 0, showing in database as null. Value should be null in PFF. This is not an issue for aggregate geographies.

Example: BK09 https://factfinder-staging.planninglabs.nyc/profile/840/economic?comparator=3&reliability=true

![image](https://user-images.githubusercontent.com/32674674/75463404-c924a100-5953-11ea-855c-993777f27c77.png)

**Comment by allthesignals** [comment]
@EricaMaurer this is a problem on production too, afaict: 

<img width="669" alt="Screen Shot 2020-03-04 at 6 54 22 PM" src="https://user-images.githubusercontent.com/5004319/75934091-9d208880-5e49-11ea-9d86-4c95ca430f87.png">

https://popfactfinder.planning.nyc.gov/profile/599/economic?reliability=true

Does this issue block the new data release from happening?

**Comment by EricaMaurer** [comment]
I'd love for it to not appear, but good to push through the new data release and tackle this at another point if it's not an easy fix.

**Comment by allthesignals** [comment]
@EricaMaurer I don't think it'll be too bad but I'd like to reduce the # of "variables" in terms of debugging. I think one thing should happen, sniff test, then the other. 

Sounds like we're cool with tackling this after.

**Comment by EricaMaurer** [comment]
Makes sense. Don't want it to get lost in the list of issues, but happy to tackle this after releasing the data.
